mp/game/momentum/resource/ui/settings/Settings_Gameplay.res

```diff
@@ -486,7 +542,7 @@
         "ControlName" "Divider"
         "fieldName"   "PaintDivider"
         "xpos" "cs-0.5"
-        "ypos" "216"
+        "ypos" "234"
```

Comment:
The spacing of these could be better:
![image](https://user-images.githubusercontent.com/9014762/90525641-52a77580-e124-11ea-88cc-d7aa3c7fb204.png)
![image](https://user-images.githubusercontent.com/9014762/90525719-694dcc80-e124-11ea-82b4-07e903ff69b9.png)

Will have to increase the height of [the tab itself](https://github.com/momentum-mod/game/blob/c775ff2d8042fbc0b5bccbec24cb6813b0413201/mp/game/momentum/resource/ui/settings/Settings_Gameplay.res#L9).